package/kernel/linux/modules/usb.mk

```diff
@@ -1274,7 +1274,8 @@ $(eval $(call KernelPackage,usb-net-smsc75xx))
 
 define KernelPackage/usb-net-smsc95xx
   TITLE:=SMSC LAN95XX based USB 2.0 10/100 ethernet devices
-  DEPENDS:=+kmod-libphy +kmod-phy-smsc +LINUX_6_1:kmod-net-selftests
+  DEPENDS:=+kmod-libphy +kmod-phy-smsc +LINUX_6_1:kmod-net-selftests \
+  	+LINUX_6_6:kmod-net-selftests
```

Comment:
Please change this dependency to `!LINUX_5_15`

Reply:
OK